staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -943,6 +943,9 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "Volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// runAsGroup is the strategy that will dictate the allowable RunAsGroup values that may be set.
+	// +optional
+	RunAsGroup *RunAsGroupStrategyOptions `json:"runAsGroup,omitempty" protobuf:"bytes,19,opt,name=runAsGroup"`
```

Comment:
AFAICT this is never being enforced?

Reply:
only the securityContext is being enforced in Container and pod spec. PSP is not being enforced and hence i added the feature gates and disabled them by default. That was the comment from @liggitt  which i have been taking care